### PR TITLE
Update where

### PR DESCRIPTION
**Pull request by jcvall**
This PR resolves #(put issue number here).

# PR Checklist

Please ensure that you have done the following:

1. [x] PR in from a fork off your branch. Do not PR from <your_username>:master, but rather from <your_username>:<branch_name>.
2. [x] If you're not on the contributors list, add yourself to `AUTHORS.rst`.

## Quick Check

To do a very quick check that everything is correct, follow these steps below:

- [x] Run the command `make check` from pyjanitor's top-level directory. This will automatically run:
    - black formatting
    - pycodestyle checkin
    - running the test suite
    - docs build

## Code Changes

If you are adding code changes, please ensure the following:

- [x] Ensure that you have added tests.
- [ ] Run all tests (`$ pytest .`) locally on your machine.
    - [x] Check to ensure that test coverage covers the lines of code that you have added.
    - [ ] Ensure that all tests pass.

## Documentation Changes

If you are adding documentation changes, please ensure the following:

- [ ] Build the docs locally.
- [ ] View the docs to check that it renders correctly.

# PR Description

Please describe the changes proposed in the pull request:

- 
- 
- 

# Relevant Reviewers

Please tag maintainers to review.

- @ericmjl

### CODE REVIEW

**Comment by jcvall** [comment]
I am done making changes.  When you get time please provide any thoughts thus far, thanks.

File: janitor/functions.py [hunk 1]
```diff
@@ -2570,3 +2570,35 @@ def find_replace(df: pd.DataFrame, column: str, mapper: dict):
...
+@pf.register_dataframe_method
+def update_where(df: pd.DataFrame, conditions: None, target_col: None, target_val: None):
+    """
+    Add multiple conditions to update a column in the dataframe.
+    Example usage:
+    
+    .. code-block:: python
+        
+        df = (
+            pd.DataFrame(...)
```
Comment (ericmjl) [inline janitor/functions.py]: @jcvall, apologies for the re-comment on the example, but I think lines 2583 and 2584 here can be omitted!

File: janitor/functions.py [hunk 2]
```diff
@@ -2570,3 +2570,35 @@ def find_replace(df: pd.DataFrame, column: str, mapper: dict):
...
+            df
+            .update_where(
+                condition=(df['column A'] == 'x') & (df['column B'] == 'y'),
+                target_col='column C',
+                target_val='z')
+            )
+            & (df['column B'] == 'y'),  
+            target_col = 'column C',
+            target_val = 'z')
+            )
```
Comment (ericmjl) [inline janitor/functions.py]: I think the same can be said for lines 2593-2596 inclusive.

**Comment by jcvall** [comment]
Thanks for the feedback.  Do you like it so far? I do but I have a bias opinion. Lol

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ericmjl** [comment]
@jcvall we're always happy to chat :smile:. 

I did a thing where I just pushed a change to the PR directly. That should address the comments I was suggesting. I noticed that you had resolved the conversations directly, but the changes requested were not reflected. That's no problem - I found a chunk of time where I could hack on the example directly, and also fix the issue causing Travis to fail.

**Comment by jcvall** [comment]
Oh thought that it changed it automatically when I hit the Button.  You have been patient with me.  I plan on taking a course on git and github.  At the moment i really am learning from trial and error.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ericmjl** [comment]
> Oh thought that it changed it automatically when I hit the Button.

Ah! I see where the misunderstanding happened. No worries! Btw, trial and error has been super effective for my learning too :smile:. You've been contributing productively, and I'm thankful for that!

**Pull request merged**